slash/react/src/Title/Client/Title.tsx

```diff
@@ -0,0 +1,60 @@
+import { useCallback, useMemo, type ReactNode } from "react";
+import { IconBg } from "../../client";
+import { getComponentClassName } from "../../Form/core";
+import { TitleSize } from "./constants";
+import { TitleWithSubtitles } from "./TitleWithSubtitles";
+
+type TitleProps = {
+  children: ReactNode;
+  firstSubtitle: string;
+  className?: string;
+  classModifier?: string;
+  icon?: ReactNode;
+  secondSubtitle?: string;
+  size?: TitleSize;
+};
+
+export const Title = ({
+  children: title,
+  className,
+  classModifier,
+  firstSubtitle,
+  icon,
+  secondSubtitle,
+  size = TitleSize.XL,
+}: TitleProps) => {
+  const componentClassName = useMemo(
+    () =>
+      getComponentClassName(
+        className,
+        classModifier ? `${classModifier} ${size}` : size,
+        "af-title",
+      ),
+    [classModifier, className, size],
+  );
+  const TitleWithSubtitlesPart = useCallback(
+    () => (
+      <TitleWithSubtitles
+        title={title}
+        firstSubtitle={firstSubtitle}
+        secondSubtitle={size === TitleSize.XL ? secondSubtitle : undefined}
```

Comment:
On peut mémoriser la condition uniquement au lieu de l'intégralité du composant :)

Reply:
Je ne vais pas faire ça mais je vais l'améliorer.
Merci de l'avoir relevé 😉

Reply:
Resolved in https://github.com/AxaFrance/design-system/pull/434/commits/40030947ca0d786804a71547b1d6e3b74bed86c5